WooCommerce/Classes/ViewRelated/Orders/Collect Payments/CollectOrderPaymentUseCase.swift

```diff
@@ -565,11 +565,28 @@ private extension CollectOrderPaymentUseCase {
                 }
             })
         }
-        // Presents receipt alert
-        alertsPresenter.present(viewModel: paymentAlerts.success(printReceipt: receiptPresentationCompletionAction,
-                                                                 emailReceipt: receiptPresentationCompletionAction,
-                                                                 noReceiptAction: { onCompleted() },
-                                                                 email: order.billingAddress?.email))
+            // Sends receipt via API
```

Comment:
It's the first time I have made some additions to the use case. Let me know if it's preferred that `ReceiptEmailView` would be a dumb view and email sending would happen within the PaymentOrchestrator rather than within `ReceiptEmailViewModel`.

Reply:
I have no specific input here, I think I've never touched those files either (or I did slightly), my 0.02: 

I think it's fine where it is now, as within `CollectOrderPaymentUseCase` we're handling the entire lifecycle from connection to payment, alerts, receipts, and analytics until is entirely completed. However, at some point we should move this out of here to some service/provider in Yosemite outside of this PR.

A related item is that currently the `CollectOrderPaymentUseCase` may be handling too much responsibility, and some of this should be extracted out (eg: alerts, hardware references, ... ).  The Orchestrator seems to be mixing some of these responsibilities as well, as is handling capture, but also receipts, which I think comes from when we were generating these entirely in the client, rather than backend-generated.